hi-ml-azure/testazure/testazure/test_datasets.py

```diff
@@ -164,7 +164,7 @@ def test_get_dataset() -> None:
     Test if a dataset that does not yet exist can be created from a folder in blob storage
     """
     # A folder with a single tiny file
-    tiny_dataset = "himl-tiny_dataset"
+    tiny_dataset = "himl_tiny_dataset"
```

Comment:
[Minor] Can we maybe add an explicit v1 here?

Reply:
Dataset is a v1 concept - in v2 they're called data assets

Reply:
Or do you mean in the name of the dataset itself? (Rather than the name of the variable)

Reply:
Would we then need to append "v1" to all dataset names?